src/style.css

```diff
@@ -0,0 +1,28 @@
+body {
+  margin: 0;
+  /* font-size: 0; */
+}
+
+/* comment .stars and uncomment the others, font-size metters there which i am curios about, why?*/
+.stars {
+  display: flex;
+  flex-flow: row wrap;
+  width: 100px;
+}
+
+.stars__star {
+  /* display: inline-block; */
```

Comment:
also, if you set `display: flex` for the parent, the `display` property for direct children is ignored. that's why you do not need it here, since `.stars__star` are direct children of `.stars`